lib/chutney/TorNet.py

```diff
@@ -764,6 +764,19 @@ def _genRouterKey(self):
             sys.exit(1)
         self._env['fingerprint'] = fingerprint
 
+
+    def _setEd25519Id(self):
+        """Make chutney check for relay microdescriptors before verifying"""
+        datadir = self._env['dir']
+        key_directory =  os.path.join(datadir, 'keys', "ed25519_master_id_public_key")
```

Comment:
What happens if the file does not exist?

Reply:
I will log out the error that either the provided path is wrong or the file does not exist.